client/src/context/useNotificationContext.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useContext, createContext, FunctionComponent, SyntheticEvent, useCallback, useEffect } from 'react';
+import { Notification } from '../interface/Notification';
+import { useAuth } from './useAuthContext';
+import getNotifications from '../helpers/APICalls/getNofitications';
+import postNotification from '../helpers/APICalls/postNotification';
+import { NotificationApiData } from '../interface/NotificationApiData';
+import { useSnackBar } from './useSnackbarContext';
+
+interface INotificationContext {
```

Comment:
```suggestion
interface NotificationContext {
```
We don't need to prefix our interfaces with 'I' in JS

Reply:
Np, changed in commit.